README.md

```diff
@@ -1,63 +1,136 @@
+- **At the home page, in mobile version, you can expand the list of Featured Singers or hide them as well using the More/Less button.**
+- **In mobile, you can access the menu pressing the Burger icon, you can close the menu using the X in top/right corner, or chosing an option from the menu.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [comming soon - Live Demo]()
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- [ ] Navigate to the button on the top right green "code" button.
+- [ ] In the drop down menu choose "Download Zip".
+- [ ] After extracting the files, you'll have my project on your local machine.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👨‍💻 sonuenginner
+
+- GitHub: [@sonuenginner](https://github.com/sonuengineer)
+- Twitter: [@sonuenginner](https://twitter.com/)
+- LinkedIn: [sonuenginner](https://linkedin.com/in/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add a couple more breakpoints to improve the responsiveness**
+- [ ] **Add a contact form, so the visitors can ask for further details.**
+- [ ] **Add pop-up cards to show more info about the singers.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+In order to improve this project, contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
 
-Give a ⭐️ if you like this project!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+I really enjoyed making this project, so, if you like it, I appreciate your support giving me a ⭐.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+This project was made using the template provided by Cindy Shin on Behance and you can view it [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Why should you use this project?**
+
+  - Because you can take a look a the basics of HTML, CSS & JS
+
+- **Why did I make this project?**
+
+  - I made this project to show everyone that I'm capable of making a website from scratch, without using any framework like BootStrap, and I know you can do it the same way as well.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 
-## Acknowledgments
+## 📝 License <a name="license"></a>
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great work @sonuengineer 🤝

- [ ] Kindly fix this MIT license link which is broken. This will make your readme file more professional since this gives users the authority to reuse your code.